lib/ical/recur_iterator.js

```diff
@@ -974,6 +972,9 @@ ICAL.RecurIterator = (function() {
             }
           }
         }
+        // Return dates in order of occurrence (1,2,3, etc...)
+        // instead of by weekday (su, mo, etc..).
+        this.days.sort(function(a,b){return a - b;});
```

Comment:
Actually it's an array of numbers, for this reason sort() needs a comparator otherwise (without comparator) it sorts alphabetically and ascending. Am I wrong?